### PR TITLE
[ja] Localize cloud native tech

### PR DESCRIPTION
**Pull request by naonishijima**
### Describe your changes
Localize cloud native tech

### Related issue number or link (ex: `resolves #issue-number`)
#2113 

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

File: content/ja/cloud-native-tech.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+## 解決すべき問題はなんですか
+
+クラウドネイティブスタックには多くの異なる技術カテゴリーがあり、様々な課題に対応しています。
+[CNCFクラウドネイティブランドスケープ](https://landscape.cncf.io/)を見ると、いかに多くの異なる領域に触れているかわかると思います。
+しかし高いレベルでは、従来のIT運用モデルの欠点という1つの主要な課題に取り組んでいます。課題としては、拡張性がありフォールトトレラントな自己修復型アプリケーションの実現が困難であること、リソースの活用が非効率であることなどが挙げられます。
+
+## どのように役に立つのでしょうか
+
+それぞれの技術は特定の問題に対応していますが、クラウドネイティブテクノロジーのグループとして、弾力性があり管理可能で観測可能な疎結合のシステムを実現します。堅牢な自動化と組み合わせることで、エンジニアは最小限の労力で影響力の大きい変更を頻繁にかつ予測通りに行うことができます。クラウドネイティブシステムの望ましい特性は、クラウドネイティブスタックで実現することが容易になりました。
```
Comment (kaitoii11) [inline content/ja/cloud-native-tech.md:20]: In my opinion, 回復性 seems to be more accurate than 弾力性。

Comment (naonishijima) [reply content/ja/cloud-native-tech.md]: Thanks for review. I will fix that.

**Comment by naonishijima** [comment]
@jihoon-seo I have a question. There is no Japanese approvers in CODEOWNERS. When creating a pull request,  PR will assigns maintainers and English approvers automatically as reviewers. Is it expected behavior? If I am wrong, please let me know the correct operation.

**Review by kaitoii11 — APPROVED** [review]

**Comment by naonishijima** [comment]
@inductor  This is a friendly reminder. Please review this PR.

**Comment by naonishijima** [comment]
@inductor  I'm sorry to bother you again. This is a friendly reminder that Please review this PR.

**Comment by jihoon-seo** [comment]
@naonishijima Sorry for late reply. After you asked, the `CODEOWNERS` file in `dev-ja` branch has been updated.
So you can see that GitHub now requests reviews to Japanese l10n approvers for Japanese l10n PRs, like #2193.

**Comment by naonishijima** [comment]
Thanks!

**Pull request closed without merging**